dips/dip-158.md

```diff
@@ -84,95 +85,86 @@ Some examples for such flows are:
 * Recurring payments
 
 ---
-## Sender / Receiver Interface
----
-This section describes a set of functionalities / activities that should be made available by a Sender and Receiver VASPs in order to support P2M Charge flow based on this DIP. Each functionality can be conceptually compared to an API endpoint. Similar to API documentation, it includes the expected input and output for each functionality.
+## Interface
+This section describes a set of functionalities / activities that should be made available by a Customer and Merchant VASPs in order to support P2M Charge flow based on this DIP. Each functionality can be conceptually compared to an API endpoint. Similar to API documentation, it includes the expected input and output for each functionality.
 
 ## Command Types
 The following is a list of new values for the `command_type` field of the `CommandRequestObject`. Each command type denotes a step in the off-chain negotiations and specifies the data of the request and response 
 * `GetPaymentInfo` - Allows obtaining payment information based on reference id
-* `InitChargePayment` - Initializes a payment process of type `charge`. Triggering this command by the wallet means the customer approved to proceed with the payment and that the wallet is satisified with the business infromation provided. Successful response from the receiver means that the receiver is satisfied with the payer data provided by the wallet 
+* `InitChargePayment` - Initializes a payment process of type `charge`. Triggering this command by the Wallet (Customer VASP) (Customer VASP) means the customer approved to proceed with the payment and that the Wallet (Customer VASP) (Customer VASP) is satisified with the business infromation provided. Successful response from the Merchant VASP means that the Merchant VASP is satisfied with the payer data provided by the Wallet (Customer VASP) (Customer VASP) 
```

Comment:
Is there an error here? Double parentheses

Reply:
Indeed find/replace error.
fixed.